app/realtime.php

```
@@ -224,6 +224,13 @@ function getTelemetry(int $workerId): Utopia\Telemetry\Adapter
     }
 }
 
+if (!function_exists('triggerStats')) {
+    function triggerStats(array $event, string $projectId): void
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$event'. (undefined)

(UnusedFormalParameter)